doc/contributing/internals.rst

```diff
@@ -0,0 +1,11 @@
+Internals
+=========
+
+This section contains information about some of the internals and
+concepts of the remoteStorage library.
+
+.. toctree::
+   :maxdepth: 2
+
+   internals/discovery-bootstrap
```

Comment:
I still don't understand that document at all. It doesn't explain what's being explained, and even I as a core developer just have no idea what it's talking about really.

Reply:
I also don't understand the Discovery bootstrap document. I don't know what "bootstrap" means in this case, it's only used in the title